YouTube-Video-Downloader/youtube_vid_dl.py

```diff
@@ -0,0 +1,28 @@
+from tkinter import *
+from pytube import YouTube
+
+root = Tk()
+root.geometry('700x300')
+root.resizable(0,0)
+root.title("YouTube Video Downloader")
+
+Label(root,text = 'Copy the link of the video you want to download from YouTube', font = 'arial 15 bold').pack()
+
+##enter link
+link = StringVar()
+
+Label(root, text = 'Paste Link Here:', font = 'arial 15 bold').place(x = 270 , y = 60)
+link_enter = Entry(root, width = 80,textvariable = link).place(x = 32, y = 90)
```

Comment:
The Entry function doesn't have any return value , why are you assigning it to a variable

Reply:
Updated!
Thank you :)